src/workerd/jsg/util.c++

```diff
@@ -204,6 +225,15 @@ DecodedException decodeTunneledException(v8::Isolate* isolate,
     setRemoteError(isolate, result.handle);
   }
 
+  if (util::Autogate::isEnabled(util::AutogateKey::ACTOR_EXCEPTION_PROPERTIES)) {
+    if (excType == kj::Exception::Type::DISCONNECTED) {
+      setRetryableError(isolate, result.handle);
+    }
+    if (excType == kj::Exception::Type::OVERLOADED) {
```

Comment:
nit: it's slightly cleaner for this to be an if/else because they are mutually exclusive.